source/clear-linux/guides/maintenance/swupd-guide.rst

```diff
@@ -1,185 +1,364 @@
+          ...50%
+      Extracting kernel-container pack for version 24430
+          ...100%
+      Starting download of remaining update content. This may take a while...
+          ...100%
+      Finishing download of update content...
+      Installing bundle(s) files...
+          ...100%
+      Calling post-update helper scripts.
+      Successfully installed 1 bundle
+
+Example 3: Verify and correct sytem file mismatch
+=================================================
+
+:command:`swupd` can determine whether system directories and files have
+been added to, overwritten, removed, or modified (e.g., permissions).
+
+.. code-block:: bash
+
+   sudo swupd verify
+
+All directories that are watched by :command:`swupd` are verified according
+to the manifest data and hash mismatches are flagged as follows:
+
+.. code-block:: console
+
+   Verifying version 23300
+   Verifying files
+      ...0%
+   Hash mismatch for file: /usr/bin/chardetect
+   ...
+   ...
+   Hash mismatch for file: /usr/lib/python3.6/site-packages/urllib3/util/wait.py
+      ...100%
+   Inspected 237180 files
+      423 files did not match
+   Verify successful
+
+In this case, python packages that were installed on top of the default
+install were flagged as mismatched. :command:`swupd` can be directed to
+ignore or fix issues based on command line options.
+
+:command:`swupd` can correct any issues it detects. Additional directives
+can be added including a white list of directories that will be ignored.
+
+The following command will repair issues, remove unknown items, and
+ignore files or directories matching `/usr/lib/python`:
+
+.. code-block:: bash
+
+   sudo swupd verify --fix --picky --picky-whitelist=/usr/lib/python
+
+Quick Reference
+***************
+
+swupd info
+   To see the currently installed version and update servers.
+
+swupd update <version number>
+   To update to a specific version or with no arguments to update to latest.
+
+swupd bundle-list [--all]
+   To list installed bundles.
+
+swupd bundle-add [-b] <search term>
+   To find a bundle that contains your search term.
+
+swupd bundle-add <bundle name>
+   To add a bundle.
+
+swupd bundle-remove <bundle name>
+   To remove a bundle.
+
+swupd --help
+   For additional :command:`swupd` commands.
+
+man swupd
+   To reference the :command:`swupd` man page, or see the 
+   `source documentation`_ available on github.
+
+Related topics
+**************
+
+* :ref:`autospec`
+* :ref:`mixer`
+* :ref:`bundles`
+
+.. _source documentation: https://github.com/clearlinux/swupd-client/blob/master/docs/swupd.1.rst
+
+.. _Kata Containers: https://clearlinux.org/containers
+
+.. _one bundle: https://github.com/clearlinux/clr-bundles/blob/master/bundles/containers-virt
+
+.. _Bundle Definition Files: https://github.com/clearlinux/clr-bundles
+
+.. _bundles: https://github.com/clearlinux/clr-bundles/tree/master/bundles
+
+.. _packages: https://github.com/clearlinux/clr-bundles/blob/master/packages
```

Comment:
This file is being deprecated. Packages are moving to the bundles directory, so I think we can just take this out.